rotkehlchen/db/addressbook.py

```diff
@@ -90,36 +90,42 @@ def get_addressbook_entries(
 
     def add_addressbook_entries(
             self,
-            book_type: AddressbookType,
+            write_cursor: 'DBCursor',
             entries: list[AddressbookEntry],
+            exists_ok: bool = False,
```

Comment:
I [told you](https://github.com/rotki/rotki/pull/5828#discussion_r1156376139) to remove this argument in the previous review unless it's absolutely needed. And I don't think it is from the way you use it now. You only should call this function for new pools.